src/Apps/Auction/routes.tsx

```diff
@@ -0,0 +1,39 @@
+// import { getRedirect } from "Apps/Order/getRedirect"
+// import { confirmRouteExit, redirects } from "Apps/Order/redirects"
+import { RouteConfig } from "found"
+import * as React from "react"
+import { graphql } from "react-relay"
+import { AuctionApp } from "./AuctionApp"
+
+// import { RegisterFragmentContainer as RegisterRoute } from "Apps/Auction/Routes/Register"
+
+// @ts-ignore
```

Comment:
Are these `@ts-ignore`s still necessary? I recall these interfaces needed to be imported for automatic TS declaration output but recent TypeScripit versions seem to have fixed this.